common/static/common/js/spec/utils/edx.utils.validate_spec.js

```diff
@@ -0,0 +1,197 @@
+;(function (define) {
+    'use strict';
+    define(['jquery', 'common/js/utils/edx.utils.validate'],
+        function($, EdxUtilsValidate) {
+
+        describe("EdxUtilsValidate", function() {
+            var fixture = null,
+                field = null,
+                result = null,
+                MIN_LENGTH = 2,
+                MAX_LENGTH = 20,
+                VALID_STRING = 'xsy_is_awesome',
+                SHORT_STRING = 'x',
+                LONG_STRING = 'xsy_is_way_too_awesome',
+                EMAIL_ERROR_FRAGMENT = 'formatted',
+                MIN_ERROR_FRAGMENT = 'least',
+                MAX_ERROR_FRAGMENT = 'up to',
+                REQUIRED_ERROR_FRAGMENT = 'Please enter your',
+                CUSTOM_MESSAGE = 'custom message';
+
+            var createFixture = function( type, name, required, minlength, maxlength, value ) {
+                setFixtures('<input id="field" type=' + type + '>');
+
+                field = $('#field');
+                field.prop('required', required);
+                field.attr({
+                    name: name,
+                    minlength: minlength,
+                    maxlength: maxlength,
+                    value: value
+                });
+            };
+
+            var expectValid = function() {
+                result = EdxUtilsValidate.validate(field);
```

Comment:
validate was attempted to be accessed via edx.utils.validate (global namespace).